unicode-data-names/unicode-data-names.cabal

```diff
@@ -0,0 +1,84 @@
+cabal-version:       2.2
+name:                unicode-data-names
+version:             0.3.0
+synopsis:            Unicode characters names and aliases
+description:
+  @unicode-data-names@ provides Haskell APIs to efficiently access the Unicode
+  character database. Performance is the primary goal in the design of
```

Comment:
to efficiently access "unicode character names from the Unicode character database" would be more appropriate. Also, I am not sure about the perf goal for this particular package we can remove that line, we may not be very strict about the perf of accessing names.

Reply:
Done